pennylane_lightning/lightning_qubit/_state_vector.py

```diff
@@ -267,10 +267,10 @@ def _apply_lightning_midmeasure(
         """
         wires = self.wires.indices(operation.wires)
         wire = list(wires)[0]
-        circuit = QuantumScript([], [qml.sample(wires=operation.wires)], shots=1)
         if postselect_mode == "fill-shots" and operation.postselect is not None:
             sample = operation.postselect
         else:
+            circuit = QuantumScript([], [qml.sample(wires=operation.wires)], shots=1)
```

Comment:
Is this line tested somewhere?

Reply:
This is tested in native MCM tests. I think this is a fluke, otherwise the next couple of lines would not be tested either.